src/olympia/users/management/commands/createsuperuser.py

```diff
@@ -80,6 +99,28 @@ def handle(self, *args, **options):
                 'api-secret': apikey.secret
             }))
 
+        if options.get('save_api_credentials', False):
+            hostname = options.get('hostname', False) or os.environ[
+                'PYTEST_BASE_URL']
```

Comment:
This can be simplified to ``options.get('hostname', os.environ.get('PYTEST_BASE_URL', False))`` which should do the same (use the `PYTEST_BASE_URL` or `False` if `options['hostname']` is not given.)